hail/src/main/scala/is/hail/expr/ir/agg/CollectAggregator.scala

```diff
@@ -9,7 +9,7 @@ import is.hail.io.{BufferSpec, InputBuffer, OutputBuffer}
 
 class CollectAggregator(val elemType: PType) extends StagedAggregator {
 
-  val resultType = PArray(elemType, required = true)
+  val resultType = PArray(elemType, required = false)
```

Comment:
Is this change just until we can handle requiredness correctly in the IR?

Reply:
yes.